Source/Core/VideoCommon/VideoCommon.h

```diff
@@ -57,6 +63,21 @@ struct TargetRectangle : public MathUtil::Rectangle<int>
 #endif
 };
 
+// This structure should only be used to represent a rectangle in EFB
+// coordinates, where the origin is at the upper left and the frame dimensions
+// are 640 x 528.
+using EFBRectangle = TaggedRectangle<struct EFBRectangleTag>;
```

Comment:
After linting, this ends up being four lines instead of one, which to me seems way too verbose for something simple like tagging. What do you think?

```c++
struct EFBRectangleTag
{
};
using EFBRectangle = TaggedRectangle<EFBRectangleTag>;
```